doc/source/whatsnew/v1.1.0.rst

```diff
@@ -901,6 +901,9 @@ I/O
 - Bug in :meth:`~DataFrame.read_feather` was raising an `ArrowIOError` when reading an s3 or http file path (:issue:`29055`)
 - Bug in :meth:`~DataFrame.to_excel` could not handle the column name `render` and was raising an ``KeyError`` (:issue:`34331`)
 - Bug in :meth:`~SQLDatabase.execute` was raising a ``ProgrammingError`` for some DB-API drivers when the SQL statement contained the `%` character and no parameters were present (:issue:`34211`)
+- :func:`pandas.read_csv` now accepts engine="arrow" as an argument, allowing for faster csv parsing
```

Comment:
move to Other Enhancements section, this should also include a link to the updated io.rst docs (do that too :->)